osu.Game/OsuGame.cs

```diff
@@ -755,6 +754,7 @@ protected override void LoadComplete()
 
             chatOverlay.State.ValueChanged += state => channelManager.HighPollRate.Value = state.NewValue == Visibility.Visible;
 
+            Add(difficultyRecommender);
```

Comment:
What's the intention behind moving this to a non-async load? It doesn't look like it will cause any issue, but does change the order at which these web requests would be fired (making them potentially much earlier in game startup).

Reply:
I saw test failures somewhere but I can't find them anymore... With songselect being entered and the recommended difficulty not being selected. May be okay to revert but isn't it better this way? I don't really see why the difficulty recommender should be loaded after a bunch of other stuff in the first place - it's not a large component.

Reply:
The intention was to delay the web requests further into the startup process (since they will be at the start of the queue on connect and may delay other more important requests from happening otherwise). But this isn't the only component to potentially do this, so I think we can try and see how it goes.